site/content/en/reference/live/apply/_index.md

```diff
@@ -36,6 +37,20 @@ group.  `kpt live apply` tracks the state of your applied resource set and
 related configuration. This helps `kpt` to reliably reconcile the real world
 resources with your configuration changes.
 
+### Client-Side Apply versus Server-Side Apply
+
+kpt live apply defaults to client-side apply, so the patches used to
+update resources are calculated on the client-side. In order to have
+the server calculate the patches, the same as `kubectl` add the
```

Comment:
Maybe rephrase the last part of this sentence.
I also don't think we need to cover SSA in detail here, we can add pointers to the kubernetes docs that describes it in detail. Mentioning the default field owner seems useful though.